rpc/server.go

```diff
@@ -68,6 +68,10 @@ func NewServer(executionPoolSize uint64, executionPoolRequesttimeout time.Durati
 	// as the services and methods it offers.
 	rpcService := &RPCService{server}
 	server.RegisterName(MetadataApi, rpcService)
+
+	// Start reporting metrics for the execution pool
+	go server.executionPool.reportMetrics(5 * time.Second)
```

Comment:
could we reuse CollectProcessMetrics or at least make `5 * time.Second` the same duration as in CollectProcessMetrics the refresh rate?

Reply:
We can surely match the duration. I didn't reuse the `CollectProcessMetrics` because it's in `metrics` package and I'd have to send the instance of execution pool in order to emit the metrics from that point. Also, it's mostly reporting CPU and memory stats.